src/pretalx/orga/templates/orga/base.html

```diff
@@ -160,50 +160,46 @@
             </div>
         {% endif %}
         <div id="page-wrapper">
-            {% if not request.user.is_anonymous %}<aside class="nav flex-column sidebar">
-                <details class="dropdown nav-link p-0" id="nav-search-wrapper">
-                    <summary id="nav-search" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                        <span id="search-context-icon" class="fa-stack fa-lg">
-                            <i class="fa fa-circle fa-stack-2x"></i>
-                            <i class="fa fa-{% if request.event %}calendar{% elif request.organiser %}group{% else %}user{% endif %} fa-stack-1x fa-inverse"></i>
-                        </span>
-                        <div id="search-context-text">
-                            {% if request.event %}
-                                <span class="context-name">{{ request.event.name }}</span>
-                                <span class="context-meta">{{ request.event.get_date_range_display }}</span>
-                            {% elif request.organiser %}
-                                <span class="context-name">{{ request.organiser.name }}</span>
-                                <span class="context-meta">{% translate "Organiser account" %}</span>
-                            {% else %}
-                                <span class="context-name">{{ request.user.get_display_name }}</span>
-                            {% endif %}
-                        </div>
-                        <div class="arrow">
-                            <i class="fa fa-angle-down"></i>
+            {% if not request.user.is_anonymous %}
+            <aside class="nav flex-column sidebar">
+                <div class="dropdown nav-link p-0" id="nav-search-wrapper">
+                    <a href="{% if request.event %}{{ request.event.orga_urls.base }}{% elif request.organiser %}{{ request.organiser.orga_urls.base }}{% else %}{% url 'orga:event.list' %}{% endif %}" style="text-decoration: none; color: inherit;">
+                        <div id="nav-search" class="dropdown-toggle summary-div" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                            <div class="d-flex justify-content-between align-items-center">
+                                <span id="search-context-icon" class="fa-stack fa-lg">
+                                    <i class="fa fa-circle fa-stack-2x"></i>
+                                    <i class="fa fa-{% if request.event %}tachometer{% elif request.organiser %}group{% else %}user{% endif %} fa-stack-1x fa-inverse"></i>
```

Comment:
Please move the complex code to Python (in Django view or in custom filter). By the way, it is a good practice to keep the class name in full form, for example, you should do:

```py
icon_class = 'fa-tachometer' if request.event else 'fa-group'
```

rather than

```py
icon_class = 'fa-' + generate()
```

Because the first form allow some tree-shaking tool to track which CSS classess are used and delete unused ones from the final CSS code delivered to users.